README.md

```diff
@@ -1 +1,49 @@
-# ansible-role-consul-alerts
\ No newline at end of file
+Ansible Role: Consul Alerts
+===========================
+
+Install Consul Alerts on Amazon EC2 or CentOS server.
+
+Requirements
+------------
+
+Written in Ansible 2.0.*
+
+Role Variables
+--------------
+
+Available variables are listed below, along with default values (see `defaults/main.yml`):
+
+### consul_alerts_port
+
+Port to bind to, default is 8700
+
+### consul_alerts_supervisor_enabled
+
+Use Supervisord for controlling Consul Alerts, default is true
+
+### consul_alerts_version
+
+Version of Consul Alerts to install, default is 0.3.0
+
```

Comment:
I followed the pattern that seems to be done on the other roles I've touch, wherein only ones that can be changed are listed. But, yes, I see your point, we said _available_ :smile_cat: